e2e/tests/localAuthorityCorrectsReturnedApplication_test.js

```diff
@@ -0,0 +1,47 @@
+const config = require('../config.js');
```

Comment:
@douglasrice please review

Reply:
e2e covers the AC scenarios for this ticket